app/modules/asset/service.server.ts

```diff
@@ -2303,11 +2303,23 @@ export async function bulkDeleteAssets({
           })
         )
       );
-    });
+    } catch {
```

Comment:
We should catch teh cause here and send it with the error, so it bubbles up to the next catch.